diff --git a/node_modules/emoji-mart/dist/main.js b/node_modules/emoji-mart/dist/main.js
index a77987e..c4c90ac 100644
--- a/node_modules/emoji-mart/dist/main.js
+++ b/node_modules/emoji-mart/dist/main.js
@@ -3213,6 +3213,7 @@ var $e3d2d32fa7bd8892$export$2e2bcd8739ae039 = /*#__PURE__*/ function(HTMLElemen
                 if (!styles) return;
                 var style = document.createElement("style");
                 style.textContent = styles;
+                style.nonce = window.__webpack_nonce__;
                 this.shadowRoot.insertBefore(style, this.shadowRoot.firstChild);
             }
         }
diff --git a/node_modules/emoji-mart/dist/module.js b/node_modules/emoji-mart/dist/module.js
index 158bd81..3a5d2b0 100644
--- a/node_modules/emoji-mart/dist/module.js
+++ b/node_modules/emoji-mart/dist/module.js
@@ -1310,6 +1310,7 @@ class $26f27c338a96b1a6$export$2e2bcd8739ae039 extends (0, $6f57cc9cd54c5aaa$exp
         if (!styles) return;
         const style = document.createElement("style");
         style.textContent = styles;
+        style.nonce = window.__webpack_nonce__;
         this.shadowRoot.insertBefore(style, this.shadowRoot.firstChild);
     }
     constructor(props, { styles: styles  } = {}){
